make/RunTests.gmk

```diff
@@ -349,13 +349,24 @@ ExpandJtregPath = \
     )) \
   )
 
+# with selection: dir/Test.java#selection -> Test.java#selection -> .java#selection -> #selection
+# or without:     dir/Test.java           -> Test.java           -> .java           -> <<empty string>>
+TestID = \
+    $(subst .java,,$(subst .java,,$(suffix $(notdir $1))))
+
+# The test selector starting with a hash (#testid) will be stripped by all
```

Comment:
Small nit - I think "selection" and "test selector" in the comments should be changed to "test id" so you don't call the same thing with 3 different names. But otherwise the change looks good to me.

Reply:
Agree and fixed.